### ISSUE TITLE
Replace generic Substack icon with real newsletter icons

### ISSUE THREAD

**Issue by jurajpal** [nicholaschiang/newsletter#11]
![image](https://user-images.githubusercontent.com/8016147/116097834-60bc7700-a65f-11eb-929f-0867aac84315.png)

**Comment by nicholaschiang** [comment]
@jurajpal Do you know where I would get those Substack icons? For example, what is the URL to the correct icon for **Not Boring** (notboring@substack.com) or **Mindy Zhang** (mindy@substack.com)?

**Comment by jurajpal** [comment]
@nicholaschiang You're right, they don't actually show up in Gmail, instead, you can see them when reading Substack in the browser (e.g.: https://www.notboring.co/ and https://mindy.substack.com/p/you-have-more-clarity-than-you-think).

Maybe I could try scraping the images/icons from the Substack website? Or can you think of a different solution?
